test/helpers.js

```diff
@@ -153,7 +153,7 @@ export function normalizeHtml(window, html) {
 export function setupHtmlEqual() {
 	const window = env();
 
-	assert.htmlEqual = (actual, expected, message) => {
+	assert.htmlEqual = (actual, expected, message) => { // eslint-disable-line no-import-assign
```

Comment:
this is the single line ignore i mentioned above. it suppresses the following warning:

```bash
../svelte/test/helpers.js
  156:2  warning  The members of 'assert' are read-only  no-import-assign
```

Reply:
do we need it?

Reply:
i'm not sure what you mean. `.htmlEqual` is certainly in use in the test suite.

https://github.com/sveltejs/svelte/search?q=htmlEqual&unscoped_q=htmlEqual